src/components/GridItem/_gridItem.scss

```diff
@@ -11,13 +11,11 @@
 
   &-align-center {
     grid-column: math.ceil(math.div($total-cols + 1 - $span, 2)) / span #{$span};
-    text-align: center;
```

Comment:
this and below is center aligning the text in every grid item by default

that should be in control of the text, not the grid item?